.github/workflows/playwright-testing.yml

```
@@ -102,16 +102,16 @@ jobs:
       - run: echo "Running Playwright PoC tests for draft PR"
 
       - name: Define node version
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
-          node-version: 20
+          node-version: 24
 
       - name: Check out repository code
         uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)